webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/BreadcrumbSwitcherToggler.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { Button, Icon } from 'patternfly-react';
 
+// eslint-disable-next-line react/prefer-stateless-function
 class BreadcrumbSwitcherToggler extends React.Component {
```

Comment:
why? can't we convert it to a stateless function?

Reply:
No, since it uses `refs` inside it must be class component.
I actually converted it to a function and then I got an error about it.

See: https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element